src/irmin-pack/unix/pack_index_intf.ml

```diff
@@ -36,7 +36,10 @@ module type S = sig
     string ->
     t
 
-  type error := [ `Index_failure of string | `Io_misc of Io.Unix.misc_error ]
+  type error :=
+    [ `Index_failure of string
+    | `Io_misc of Io.Unix.misc_error
+    | `Ro_not_allowed ]
```

Comment:
Since `v` does not use `Ro_not_allowed` it might be nice to not include it in its error type but I'm not sure of good names to distinguish the two categories of error. My first thought was `create_error` and `write_error` but write seems a little too broad. What do you think?

Reply:
`write_error` sounds fine, or we could do `create_error` and keep `error` (which is even more generic :smile:)

Reply:
kk, lets go with `write_error`